### PR TITLE
Initial TOC for Linux101

### PR DESCRIPTION
**Pull request by seisman**

### CODE REVIEW

File: source/computer/linux101.rst [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
 ==========
+
+:本节贡献者:
+:最近更新日期:
+:预计花费时间:
+
+----
+
+命令行与终端
+------------
```
Comment (core-man) [inline source/computer/linux101.rst:11]:
```suggestion
命令行
-------
```

File: source/computer/linux101.rst [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+
+命令行与图形界面
+^^^^^^^^^^^^^^^^
+
+打开终端
+^^^^^^^^
```
Comment (core-man) [inline source/computer/linux101.rst:17]:
```suggestion
终端
^^^^^
```

File: source/computer/linux101.rst [hunk 3]
```diff
@@ -1,2 +1,55 @@
...
+
+Linux 文件与目录管理
+--------------------
```
Comment (core-man) [inline source/computer/linux101.rst:20]:
```suggestion
文件管理
---------
```

File: source/computer/linux101.rst [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+
+熟悉 Linux 文件系统
+^^^^^^^^^^^^^^^^^^^
+
+操作文件与目录
+^^^^^^^^^^^^^^
+
+相对路径与绝对路径
+^^^^^^^^^^^^^^^^^^
```
Comment (core-man) [inline source/computer/linux101.rst:29]:
```suggestion
文件路径
^^^^^^^^^
```

File: source/computer/linux101.rst [hunk 5]
```diff
@@ -1,2 +1,54 @@
...
+
+管道
+^^^^
+
+通配符
+^^^^^^
+
+正则表达式
+^^^^^^^^^^
+
```
Comment (core-man) [inline source/computer/linux101.rst:48]:
```suggestion

Shell 脚本
------------

```

File: source/computer/linux101.rst [hunk 6]
```diff
@@ -1,2 +1,54 @@
...
+环境变量
+--------
+
```
Comment (core-man) [inline source/computer/linux101.rst:51]: Don't environmental variables belong to Shell?

Comment (seisman) [reply source/computer/linux101.rst:51]: Maybe, but I feel that environmental variables are more important so a new subsection.

File: source/computer/linux101.rst (hunk 5 above)
Comment (seisman) [reply source/computer/linux101.rst:48]: Not sure if we really want to talk about shell scripts here. We have a section for bash in the programming chapter.

File: source/computer/linux101.rst (hunk 6 above)
Comment (core-man) [reply source/computer/linux101.rst:51]: - `PATH`
- `~/.bashrc`
- `/etc/hosts`

right?

File: source/computer/linux101.rst (hunk 5 above)
Comment (core-man) [reply source/computer/linux101.rst:48]: Yes, I forgot that. Let's remove it.

File: source/computer/linux101.rst (hunk 6 above)
Comment (seisman) [reply source/computer/linux101.rst:51]: Yes for PATH and bashrc. 

I don't understand why you add /etc/hosts here.

Comment (core-man) [reply source/computer/linux101.rst:51]: We may use it to rename our computer, right?

File: source/computer/linux101.rst [hunk 7]
```diff
@@ -1,2 +1,54 @@
 Linux 入门
 ==========
+
+:本节贡献者:
+:最近更新日期:
+:预计花费时间:
+
```
Comment (core-man) [inline source/computer/linux101.rst:7]:
```suggestion


.. warning::

   本章正在编写中，尚未完成。

```

File: source/computer/linux101.rst (hunk 6 above)
Comment (core-man) [reply source/computer/linux101.rst:51]: If we use a server, we may add server IP and name in this file/

Comment (seisman) [reply source/computer/linux101.rst:51]: It makes no difference for users, so I don't think it belongs to 101.

**Comment by core-man** [comment]
I am okay with this TOC. Let's work on it.

**Review by core-man — APPROVED** [review]

**Pull request merged**